build/buildConfig.js

```diff
@@ -194,7 +194,8 @@ module.exports = (...args) => mapArgumentsToObject(args, ({
             jsonix: '@boundlessgeo/jsonix',
             // next libs are added because of this issue https://github.com/geosolutions-it/MapStore2/issues/4569
             proj4: '@geosolutions/proj4',
-            "react-joyride": '@geosolutions/react-joyride'
+            "react-joyride": '@geosolutions/react-joyride',
+            "geotiff-remote": path.join(paths.base, "node_modules", "geotiff", "dist-module", "source", "remote")
```

Comment:
Yes, I did check.
`fromArrayBuffer` still doesn't handle `signal` parameter correctly. It is lost when the passed down to `fromSource`. Reason it cannot handled with axios is because `fromSource` makes bigtiff parsing logic to read tif data and converts to image object with required functions necessary for extracting metadata informations. 

Maybe we can cancel the promise of `fromUrl` and `getImage` but not the actual fetch request action (which is a compromise) and in the process we can avoid the hack and also this will not impact user performance. Kindly let me know if this okay? @offtherailz